libs/features/icons/icons.js

```diff
@@ -55,13 +65,22 @@ export default async function loadIcons(icons, config) {
   const iconSVGs = await fetchIcons(config);
   if (!iconSVGs) return;
   icons.forEach(async (icon) => {
-    const { classList } = icon;
-    if (classList.contains('icon-tooltip')) decorateToolTip(icon);
-    const iconName = icon.classList[1].replace('icon-', '');
+    const iconNameInitial = icon.classList[1].replace('icon-', '');
+    let iconName;
+
+    if (iconNameInitial === 'tooltip') {
+      iconName = 'info';
+    } else if (iconNameInitial.includes('tooltip-')) {
+      iconName = iconNameInitial.replace(/tooltip-/, '');
+    } else {
+      iconName = iconNameInitial;
+    }
```

Comment:
I'm not a big fan of if else conditions tbh, I'd rather go for something such as this:
```suggestion
    let iconName = iconNameInitial === 'tooltip' ? 'info' : 'iconNameInitial
    if (iconNameInitial.includes('tooltip-') iconName = iconNameInitial.replace(/tooltip-/, '');
```

Reply:
Should we have 1-2 more unit tests to also capture the different branches? The icon name seems to be quite important, so we should write unit tests to ensure all those conditions do what we want them to